testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/DPoPTest.java

```diff
@@ -431,7 +431,7 @@ public void testDPoPDisabledOnUserInfo() throws Exception {
 
             // delete DPoP proof
             oauth.dpopProof(null);
```

Comment:
I think this is not needed now. I was thinking to removed this method and the property completely from `OAuthClient` but it is still used in `doPushedAuthorizationRequest`. So we have to wait.

Reply:
Yeah, I plan to completely remove this method, so will clean that up then

Reply:
Deleted

I plan to remove this field completely from OAuthClient and the method; but yes there's a few requests that needs to be updated first for them to support setting it on the request.